FAQ.md

```diff
@@ -51,97 +55,100 @@ types of programming languages.
 
 ## Licensing
 
-You are free to use Slint under either [GNU GPLv3](./LICENSES/GPL-3.0-only.txt) or 
-a [commercial license](./LICENSES/LicenseRef-Slint-commercial.md). We also offer a completely 
-free-of-charge commercial license under the [ambassador program](https://slint-ui.com/ambassador-program.html).
+Slint is available under either a [proprietary license](LICENSES/LicenseRef-Slint-commercial.md)
+or [GNU GPLv3](LICENSES/GPL-3.0-only.txt)
 
-### What are the commercial licensing options?
+### Can I license my code under a more permissive license than GPL?
 
-We offer - Ambassador, Flex and Buyout commercial licensing options. All three 
-options include a perpetual right to use Slint to develop applications, unlimited 
-distribution of such applications on desktops and [standard support](https://slint-ui.com/support/slint_support_service_agreement).
+Yes. You can publish your own source code under a permissive license compatible with
+the GPL, such as BSD, MIT, or Apache license. However, the binary or the package
+containing Slint needs to be licensed under GPL. For more info, you can check the following GPL FAQs:
 
-- The Ambassador option is a free license that can be provided in lieu of helping
-us promote Slint. Apart from the standard inclusiosn, unlimited User-Seats, updates and unlimited distribution 
-of Slint based applications on embedded devices are included.
+- [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL)
+- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)
 
-- With the Flex option, you can purchase the required number of User-Seats needed to
-develop your applications with Slint. In addition to the standard inclusions, updates are included as long 
-as the subscription is active.
+### What are the different proprietary licensing options?
```

Comment:
I wouldn't put the pricing in the FAQ in the repository, just a link to ir.

Reply:
Replaced with link